lib/portage/util/cpuinfo.py

```diff
@@ -48,3 +50,22 @@ def get_cpu_count():
         return multiprocessing.cpu_count()
     except (ImportError, NotImplementedError):
         return None
+
+
+def makeopts_to_job_count(makeopts):
+    """
+    Parse the job count (-jN) from MAKEOPTS. Python version of
+    bin/isolated-functions.sh's ___makeopts_jobs().
+
+    @return: Number of jobs to run or number of CPUs if none set.
+    """
+    if not makeopts:
+        return get_cpu_count()
+
+    # TODO: Fix
```

Comment:
This TODO is just because I think the regex could be better like the one at __makeopts_jobs. Suggestions welcome!

https://github.com/gentoo/portage/blob/master/bin/isolated-functions.sh#L472